readthedocs/doc_builder/director.py

```diff
@@ -291,6 +291,33 @@ def build_epub(self):
         return False
 
     def run_build_job(self, job):
+        """
+        Run a command specified by the user under `build.jobs.` config key.
+
+        It uses the "VCS environment" for pre_/post_ checkout jobs and "build
+        environment" for the rest of them.
+
+        Note that user's commands:
+
+        - are not escaped
+        - are run with under the path where the repository was cloned
+        - are run as RTD_DOCKER_USER user
+        - users can't run commands as `root` user
+        - all the user's commands receive environment variables
```

Comment:
These are not passed on PR builds I assume? Nothing changed there.

Reply:
Yeah, I haven't changed anything regarding env variables and pull requests. So, env variables are not passed on external versions.

I don't want to repeat that logic in the comment here, so maybe I can add modify it as "all the user's commands receive the same environment variables as regular commands" or similar.